test/benchee/benchmark_test.exs

```diff
@@ -4,18 +4,19 @@ defmodule Benchee.BenchmarkTest do
   alias Benchee.Configuration
   alias Benchee.Benchmark.{Scenario, ScenarioContext}
   alias Benchee.Test.FakeBenchmarkPrinter, as: TestPrinter
-  alias Benchee.Test.FakeBenchmarkRunner,  as: TestRunner
+  alias Benchee.Test.FakeBenchmarkRunner, as: TestRunner
   alias Benchee.Suite
 
   describe ".benchmark" do
     test "can add jobs with atom keys but converts them to string" do
-      suite = %Suite{}
-              |> Benchmark.benchmark("one job", fn -> 1 end)
-              |> Benchmark.benchmark(:something, fn -> 2 end)
+      suite =
+        %Suite{}
+        |> Benchmark.benchmark("one job", fn -> 1 end)
+        |> Benchmark.benchmark(:something, fn -> 2 end)
 
-      job_names = Enum.map(suite.scenarios, fn(scenario) -> scenario.job_name end)
+      job_names = Enum.map(suite.scenarios, fn scenario -> scenario.job_name end)
```

Comment:
:cry: 

I've seen this and I don't like it. The one place I like to use parentheses and it's taken away from me... grml.

Reply:
Yeah, this is my least favorite thing as well. I'm generally a fan of parens in all cases, but c'est la vie.